Remove sku peer dependency (#1620)
seek-oss/braid-design-system: README.md

$ pnpm storybook
```

## Building with Braid

### Styling with [Vanilla Extract]

Braid uses Vanilla Extract for styling, enabling the authoring of CSS in TypeScript in a way that can be statically extracted at build time, generating reusable atomic CSS classes.

This requires use of a bundler plugin to collect the extracted styles (see Vanilla Extract&rsquo;s [integration] documentation), either injecting them into the document or a separate CSS stylesheet.

At SEEK this is done via [sku] as part of the build process.

[Vanilla Extract]: https://vanilla-extract.style/
[integration]: https://vanilla-extract.style/documentation/getting-started

### Assertions

> To ensure correct usage of it&rsquo;s components, Braid performs some precondition and invariant checking at runtime using the [assert] library.

Reviewer: ```suggestion
To ensure correct usage of its components, Braid performs some precondition and invariant checking at runtime using the [assert] library.
```
